test/src/test/java/org/corfudb/infrastructure/ServerContextBuilder.java

```diff
@@ -46,6 +47,7 @@
     String handshakeTimeout = "10";
     String prefix = "";
     String retention = "1000";
+    String compressionCodec = Codec.Type.LZ4.toString();
```

Comment:
why we need it here? Why not use an enum instead of a string?

Reply:
Bet.

Reply:
NO
It can break your system:
1. ServerContext is a god object
2. lack of incapsulation which can easily break your system or anyone who is not very careful can break your system by dynamically updating some parameter in the ServerContext